package/src/library/DatePicker/index.tsx

```diff
@@ -0,0 +1,178 @@
+'use client'
+
+import Calendar from '../Calendar'
+import Input, { InputIconProps, InputRootProps } from '../Input'
+
+import { useState, ComponentProps, useEffect, useRef } from 'react'
+import { DayPicker, DayModifiers } from 'react-day-picker'
+
+export enum DatePickerSelectionMode {
+  Single = 'single',
+  Range = 'range',
+  Multiple = 'multiple',
+}
```

Comment:
O proprio Calendar não traz isso? Se ele não traz, deveria, tem que passar esses types pra lá se for o caso